$MirOS: ports/mailnews/alpine/patches/patch-imap_src_c-client_utf8_c,v 1.1 2008/05/07 22:28:40 tg Exp $
--- imap/src/c-client/utf8.c.orig	2008-06-04 20:18:34.000000000 +0200
+++ imap/src/c-client/utf8.c	2009-02-02 20:56:01.000000000 +0100
@@ -114,6 +114,8 @@ static const struct utf8_eucparam ksc_pa
 static const CHARSET utf8_csvalid[] = {
   {"US-ASCII",CT_ASCII,CF_PRIMARY | CF_DISPLAY | CF_POSTING,
    NIL,NIL,NIL},
+  {"ISO_646.IRV:1991",CT_ASCII,CF_PRIMARY | CF_DISPLAY | CF_POSTING,
+   NIL,NIL,NIL},
   {"UTF-8",CT_UTF8,CF_PRIMARY | CF_DISPLAY | CF_POSTING,
    NIL,SC_UNICODE,NIL},
   {"UTF-7",CT_UTF7,CF_PRIMARY | CF_POSTING | CF_UNSUPRT,
@@ -297,6 +299,8 @@ static const CHARSET utf8_csvalid[] = {
      NIL,NIL,"US-ASCII"},
   {"IBM437",CT_1BYTE,CF_PRIMARY | CF_DISPLAY,
      (void *) ibm_437tab,SC_LATIN_1,"ISO-8859-1"},
+  {"CP437",CT_1BYTE,CF_PRIMARY | CF_DISPLAY,
+     (void *) ibm_437tab,SC_LATIN_1,"ISO-8859-1"},
   {"IBM737",CT_1BYTE,CF_PRIMARY | CF_DISPLAY,
      (void *) ibm_737tab,SC_GREEK,"ISO-8859-7"},
   {"IBM775",CT_1BYTE,CF_PRIMARY | CF_DISPLAY,
